### DISCUSSION TITLE
How to prevent fans ramping for very low loading ?

### DISCUSSION THREAD

**Discussion by alex3xelax** [Rem0o/FanControl.Releases#880] [Q&A]
hi all
![fc](https://user-images.githubusercontent.com/88495460/174321799-0f519fd1-55b4-4231-bf0b-704d9c044b98.png)

I use the above fans setting for my gpu, the problem is when i use my browser. Some time i have a temp spike like 50°C for few seconds so the fans start spinning.

I liked avoid **the fans spin  when the temp hit the 50°C for few seconds** and come back quickly.

I have already setting hysteresis e incresed the response time (8s), but not fixed. 
Is there any tweak can i try ?

thanks

**Comment by Rem0o** [comment]
Remove the point on the very far left.

**Reply by alex3xelax** [reply]
thanks, im going to testing.

**Reply by alex3xelax** [reply]
hi, 

I tried remove the last left point but not fixed.
Is there a "method" to do **NOT change fans speed IF the temp spike keep for few seconds** ?
I think this is the point.

**Reply by Rem0o** [reply]
Well yes, that’s response time. Make sure hysteresis value is also high enough.

**Reply by alex3xelax** [reply]
The hysteresis/response time is 15°C/8sec.
What values can i try ?

**Reply by Rem0o** [reply]
Well that's plenty enough. That means your graph won't apply a new value until a 15 degree change is observed over a period of 8 seconds. That should be plenty to cover small spikes. Make sure you don't have the "only on the way down" checkbox checked though.
